src/components/Sort/Sort.tsx

```diff
@@ -4,7 +4,7 @@ import styles from '@/components/Sort/Sort.module.css';
 
 const Sort = () => (
     <div className={styles.sortBlock}>
-        <p className={styles.sortText}>Sort by :</p>
+        <p className={styles.sortText}>Sort by:</p>
         <select className={styles.sortSelect}>
             <option className={styles.sortSelectOption}>Price (High - Low)</option>
```

Comment:
Could you add all items there

![image](https://github.com/DavidTereshchenko/fe-react-2024/assets/108006836/346c32c5-e3d9-44d0-9750-8330c48d89c9)